packages/geany.rb

```diff
@@ -41,12 +46,16 @@ def self.install
   end
 
   def self.postinstall
+    # This is needed to aoid "Error loading theme icon 'geany-build' for stock" messages.
+    if File.exist?("#{CREW_PREFIX}/bin/gtk-update-icon-cache")
+      system "gtk-update-icon-cache #{CREW_PREFIX}/share/icons/hicolor -f"
+    end
     puts
     puts "To get started, type 'geany'.".lightblue
     puts
-    puts "To completely uninstall, execute the following:".lightblue
-    puts "crew remove geany".lightblue
-    puts "rm -rf ~/.config/geany".lightblue
+    puts 'To completely uninstall, execute the following:'.lightblue
+    puts 'crew remove geany'.lightblue
+    puts 'rm -rf ~/.config/geany'.lightblue
```

Comment:
Could these steps be added to `def self.remove` with a prompt instead?

Reply:
Yes, that could be handled in such a manner. If I get to it before I leave tomorrow I will add that.

Reply:
Added self.remove